### ISSUE TITLE
 whether tflite can not support the two operator ( MaxPool2D and QUANTIZE) in TensorFlow 2.0.0 when running the mode in embed freeRTOS system?

### ISSUE THREAD

**Issue by raygong1207** [tensorflow/tensorflow#34608]
**System information**
- OS Platform and Distribution : Window 7
- TensorFlow installed from : Install TensorFlow 2.0.0  by anaconda. It should be binary
- TensorFlow version (or github SHA if from source): 2.0.0

**Provide the text output from tflite_convert**
Below is the tf.lite.convert output information after convert finished.
```
2019-11-26 16:16:17.608537: I tensorflow/core/grappler/devices.cc:60] Number of eligible GPUs (core count >= 8, compute capability >= 0.0): 0 (Note: TensorFlow was not compiled with CUDA support)
2019-11-26 16:16:17.609537: I tensorflow/core/grappler/clusters/single_machine.cc:356] Starting new session
2019-11-26 16:16:17.712547: I tensorflow/core/grappler/optimizers/meta_optimizer.cc:716] Optimization results for grappler item: graph_to_optimize
2019-11-26 16:16:17.712547: I tensorflow/core/grappler/optimizers/meta_optimizer.cc:718]   function_optimizer: function_optimizer did nothing. time = 8.001ms.
2019-11-26 16:16:17.712547: I tensorflow/core/grappler/optimizers/meta_optimizer.cc:718]   function_optimizer: function_optimizer did nothing. time = 0ms.
2019-11-26 16:16:17.738550: I tensorflow/core/grappler/devices.cc:60] Number of eligible GPUs (core count >= 8, compute capability >= 0.0): 0 (Note: TensorFlow was not compiled with CUDA support)
2019-11-26 16:16:17.739550: I tensorflow/core/grappler/clusters/single_machine.cc:356] Starting new session
2019-11-26 16:16:17.807557: I tensorflow/core/grappler/optimizers/meta_optimizer.cc:716] Optimization results for grappler item: graph_to_optimize
2019-11-26 16:16:17.808557: I tensorflow/core/grappler/optimizers/meta_optimizer.cc:718]   constant folding: Graph size after: 15 nodes (-4), 18 edges (-4), time = 52.005ms.
2019-11-26 16:16:17.808557: I tensorflow/core/grappler/optimizers/meta_optimizer.cc:718]   constant folding: Graph size after: 15 nodes (0), 18 edges (0), time = 1ms.
INFO: Initialized TensorFlow Lite runtime.
generate: ./cnn.tflite
```

Also, please include a link to a GraphDef or the model if possible.
My first model code:
model = tf.keras.models.Sequential()
model.add(tf.keras.layers.Conv2D(filters=32,   kernel_size=[5, 5],  padding='same',
                                activation=tf.nn.relu,input_shape=(feature_cnt_max, feature_dim_cnt,1)))
model.add(tf.keras.layers.Dropout(0.2))
model.add(tf.keras.layers.MaxPool2D(pool_size=[2, 2], strides=2))
model.add(tf.keras.layers.Conv2D(filters=32, kernel_size=[3, 3],padding='same',activation=tf.nn.relu))
model.add(tf.keras.layers.MaxPool2D(pool_size=[2, 2], strides=2))
model.add(tf.keras.layers.Dropout(0.2))
model.add(tf.keras.layers.Flatten())
model.add(tf.keras.layers.Dense(units=num_fc_hide,activation=tf.nn.relu))
model.add(tf.keras.layers.Dropout(0.2))
model.add(tf.keras.layers.Dense(units=num_class,activation=tf.nn.softmax))
model.build(input_shape=(None,feature_cnt_max, feature_dim_cnt,1))

My second model code:
model = tf.keras.models.Sequential()
model.add(tf.keras.layers.Conv2D(filters=186,
                                kernel_size=[first_filter_height, first_filter_width],
                                padding='valid',
                                activation=tf.nn.relu,input_shape=(feature_cnt_max, feature_dim_cnt,1)))
model.add(tf.keras.layers.Dropout(0.2))
model.add(tf.keras.layers.Flatten())
model.add(tf.keras.layers.Dense(units=num_class,activation=tf.nn.softmax))
model.build(input_shape=(None,feature_cnt_max, feature_dim_cnt,1))

**Any other info / logs**
I convert the two model into *tflite by below code:
(https://github.com/tensorflow/tensorflow/files/3890814/covert_h5_to_tflite.txt)

covert_h5_to_tflite.py
My question is:
1: When i use the first model and do not do quantization, i meet the the err info report "Didn't find op for builtin opcode 'MaxPool2D' version '1'" when run the the model in my Embedded freeRTOS system.
2: When i use the second model and do quantization, i meet the the err info report "Didn't find op for builtin opcode 'QUANTIZE' version '1'" when run the the model in my Embedded freeRTOS system.
3: Below code add the TF_OPS, but the tflite are same with no TF_OPS, why?
    converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS,tf.lite.OpsSet.SELECT_TF_OPS]

So i want to know whether tflite can not support the two operator ( MaxPool2D and QUANTIZE) in tensorflow 2.0.0? How about tensorflow 1.1x?

**Comment by raygong1207** [comment]
The latest status is that: i can do inference in my embed board when use float32 model( just do not quantization).  but will report below error info from board when run the quantization model by TFLiteConverter. So will i close this issue and report a new one? or going on this issue?

**Comment by ninn55** [comment]
@raygong1207 
Does anyone have a fix for this?
I'm experiencing the same issue. The float model works fine. But extremely slow.
I would prefer a quantized tflite model running on tflite 2.0.0 if possible.

Thanks in advance.

**Comment by sushreebarsa** [comment]
@raygong1207 It looks like you are using an older Version of Tensorflow (2.0). Many bugs have been fixed in the latest version. Can you please execute your code using Latest Version (2.6.0) and let us know if the issue still persists? Thanks!
